ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/PrivacyBlockProcessor.java

```diff
@@ -95,31 +118,97 @@ public Result processBlock(
                             .getPrivateTransaction()
                             .getPrivacyGroupId()
                             .get());
-                if (!preProcessPrivacyGroupHeadBlockMap.containsKey(privacyGroupId)) {
+
+                final List<PrivateTransactionWithMetadata> actualList =
+                    createActualList(
+                        blockHeader, privateTransactionWithMetadataList, privacyGroupId);
+
+                if (actualList.size() > 0) {
+                  LOG.info(
```

Comment:
Maybe this should be a debug or trace level?

Reply:
Debug now!